test/sequence/dynamic_registration_test.rb

```diff
@@ -0,0 +1,42 @@
+require File.expand_path '../../test_helper.rb', __FILE__
+
+class DynamicRegistrationSequenceTest < MiniTest::Unit::TestCase
+
+  REQUEST_HEADERS = { 'Accept'=>'application/json+fhir', 
+                      'Accept-Charset'=>'UTF-8', 
+                      'Content-Type'=>'application/json+fhir;charset=UTF-8'
+                     }
+
+  RESPONSE_HEADERS = {"content-type"=>"application/json"}
+
+  def setup
+    instance = TestingInstance.new(url: 'http://www.example.com', 
+                                   client_name: 'Crucible Smart App',
+                                   base_url: 'http://localhost:4567',
+                                   client_endpoint_key: SecureRandomBase62.generate(32),
+                                   oauth_register_endpoint: 'http://oauth_reg.example.com/register',
```

Comment:
magic string could be a constant since you may be using this in multiple tests... e.g. `test_all_pass` below.